server/src/test/scala/actors/FlightMessageConversionSpec.scala

```diff
@@ -2,93 +2,45 @@ package actors
-    val iata = "BA1111"
-    val origin = PortCode("JFK")
-    val pcpTime = 10L
-    val scheduledTime = 1L
-    val estimatedTime = 2L
-    val touchdownTime = 3L
-    val estimatedChoxTime = 4L
-    val actualChoxTime = 5L
-    val feedSources = Set("ACL", "Live")
-    val transPax = 10
-
-    val carrierScheduled = 4L
-
-    val flightMessage = new FlightMessage(
-      operator = Option(operator),
-      gate = Option(gate),
-      stand = Option(stand),
-      status = Option(status),
-      maxPax = Option(maxPax),
-      actPax = Option(actPax),
-      tranPax = Option(transPax),
-      runwayID = Option(runwayId),
-      baggageReclaimId = Option(baggageReclaimId),
-      airportID = Option(airportId.iata),
-      terminal = Option(terminal.toString),
-      iCAO = Option(icao),
-      iATA = Option(iata),
-      origin = Option(origin.iata),
-      pcpTime = Option(pcpTime),
-      scheduled = Option(scheduledTime),
-      estimated = Option(estimatedTime),
-      touchdown = Option(touchdownTime),
-      estimatedChox = Option(estimatedChoxTime),
-      actualChox = Option(actualChoxTime),
-      feedSources = feedSources.toSeq,
-      carrierScheduled = Option(carrierScheduled)
-    )
-
-    val arrival = FlightMessageConversion.flightMessageToApiFlight(flightMessage)
 
+  "Given an Arrival" >> {
     import drt.server.feeds.Implicits._
 
-    val expected = Arrival(
-      Operator = Option(Operator(operator)),
-      Status = status,
-      Estimated = Option(estimatedTime),
-      Actual = Option(touchdownTime),
-      EstimatedChox = Option(estimatedChoxTime),
-      ActualChox = Option(actualChoxTime),
-      Gate = Option(gate),
-      Stand = Option(stand),
-      MaxPax = Option(maxPax),
-      ActPax = Option(actPax),
-      TranPax = Option(transPax),
-      RunwayID = Option(runwayId),
-      BaggageReclaimId = Option(baggageReclaimId),
-      AirportID = airportId,
-      Terminal = terminal,
-      rawICAO = icao,
-      rawIATA = iata,
-      Origin = origin,
-      Scheduled = scheduledTime,
-      PcpTime = Option(pcpTime),
-      FeedSources = feedSources.map(fs => FeedSource(fs)).collect { case Some(fs) => fs },
-      CarrierScheduled = Option(carrierScheduled)
-    )
-
-    arrival === expected
+    val arrival = Arrival(
+      Operator = Option(Operator("BA")),
+      Status = "landed",
+      Estimated = Option(2L),
+      Actual = Option(3L),
+      EstimatedChox = Option(4L),
+      ActualChox = Option(5L),
+      Gate = Option("G1"),
+      Stand = Option("S1"),
+      MaxPax = Option(350),
+      ActPax = Option(122),
+      TranPax = Option(10),
+      RunwayID = Option("R1"),
+      BaggageReclaimId = Option("B1"),
+      AirportID = PortCode("LHR"),
+      Terminal = T1,
+      rawICAO = "BAA1111",
+      rawIATA = "BA1111",
+      Origin = PortCode("JFK"),
+      Scheduled = 1L,
+      PcpTime = Option(10L),
+      FeedSources = Set(AclFeedSource, LiveFeedSource),
+      CarrierScheduled = Option(4L)
```

Comment:
should we have a version of this test with a suffix in it?

Reply:
yes good point. the bug i make caused this test to fail even without a suffix, but clearly we should test it with on too!